modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// configuring-a-workspace-using-a-devfile
+
+[id="allow-users-to-define-workspace-deployment-labels-and-annotations_{context}"]
+= Allow users to define workspace deployment labels and annotations
```

Comment:
```suggestion
= Allowing users to define workspace deployment labels and annotations
```

Reply:
Also, I suggest renaming the file itself (I don't think I can suggest it in GitHub through Suggestions =\)

Reply:
done